internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -2015,4 +2015,23 @@ private ConfigKeys() {
    * with SOS, EOS or skipped records.
    */
   public static final String SERVER_INGESTION_HEARTBEAT_INTERVAL_MS = "server.ingestion.heartbeat.interval.ms";
+
+  /**
+   * The size of the thread pool for AsyncGauge metric value measurement.
+   */
+  public static final String ASYNC_GAUGE_MEASUREMENT_THREAD_COUNT = "async.gauge.measurement.thread.count";
+
+  /**
+   * The maximum timeout in milliseconds for AsyncGauge metric value measurement.
+   * If this timeout is reached, the measurement will be canceled, a new measurement task will be submitted, and the
+   * cached metric value from the previous measurement (if any; otherwise 0) will be returned.
+   */
+  public static final String ASYNC_GAUGE_MEASUREMENT_MAX_TIMEOUT_IN_MS = "async.gauge.measurement.max.timeout.in.ms";
+
+  /**
+   * Whenever a new AsyncGauge metric value measurement task is submitted, the caller thread will wait for at least
+   * this amount of time before returning the cached metric value from the previous measurement (if any; otherwise 0).
+   */
+  public static final String ASYNC_GAUGE_MEASUREMENT_INITIAL_TIMEOUT_IN_MS =
+      "async.gauge.measurement.initial.timeout.in.ms";
```

Comment:
I only saw these config definitions, so which classes would pick up the configured values? I guess it will be in the LinkedIn wrapper layer, right?
If these are only used in the LinkedIn wrapper layer, shall we move these definitions into the wrapper layer instead?

Reply:
Yeah, will be used in LI wrapper only. Sure; removed them.